### PR TITLE
Navigation

### PR DESCRIPTION
**Pull request by rahulcs754**
Create Simple navigation for desktop
image
****

Please Review

### CODE REVIEW

File: css/components/navigation.css [hunk 1]
```diff
@@ -0,0 +1,66 @@
...
+}
+
+.header-nav {
+  display: flex;
+  margin-top: 5px;
+}
+
+.header-logo a,
+.header-nav a {
+  padding: 10px 15px;
```
Comment (shubhamkumar648) [inline css/components/navigation.css]: can you change in to rem

**Review by shubhamkumar648 — COMMENTED** [review]
consider comment REAMING GOOD

File: css/components/modal.css [hunk 2]
```diff
@@ -24,6 +24,21 @@
 }
 
 /* small || medium | large content */
+<<<<<<< HEAD
```
Comment (ttahm3d) [inline css/components/modal.css]: Delete this line. This might have come in while resolving merge conflicts

File: css/components/modal.css [hunk 3]
```diff
@@ -37,6 +52,7 @@
 .modal-l {
   width: 60%;
   max-width: 80%;
+>>>>>>> 0d92806528a4f2a3a07ab9856475690919106687
```
Comment (ttahm3d) [inline css/components/modal.css]: same here

**Review by ttahm3d — APPROVED** [review]
Looks good. just remove the extra lines added.

**Review by shubhamkumar648 — APPROVED** [review]
good to merge

**Pull request merged**